riemann/setup.py

```diff
@@ -49,11 +49,10 @@
 
 def configuration(parent_package='',top_path=None):
     from numpy.distutils.misc_util import Configuration
-    import numpy
 
     config = Configuration('riemann', parent_package, top_path)
 
-    src_dir = os.path.join(os.path.dirname(__file__),'')
+    src_dir = os.path.join(os.path.dirname(__file__),'../src/')
```

Comment:
Is this relative path reference robust?  Seems like at this level it would be better to construct a full-proof path to `../src/` but perhaps I am being paranoid.

Reply:
I don't understand -- how would a full path be more robust?  Feel free to propose a patch that would improve this.

If you start renaming or moving directories, the whole clawpack build will definitely break.  But that's true for any Python package that also needs to compile code, I think.

Reply:
I was thinking maybe there was a better way to do `..` but apparently not.  We could use `os.path.join("..", "src")` instead but that's not all that important.

Reply:
Ah, I guess you're right -- that would at least be more platform-independent.  I will do that.